projects/admin/src/app/service/record-routing.service.ts

```diff
@@ -0,0 +1,392 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2019 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Injectable } from '@angular/core';
+import { Router, UrlSegment } from '@angular/router';
+import { RecordSearchComponent, DetailComponent, EditorComponent, ActionStatus } from '@rero/ng-core';
+import { CirculationPolicyComponent } from '../record/custom-editor/circulation-settings/circulation-policy/circulation-policy.component';
+import { CircPoliciesBriefViewComponent } from '../record/brief-view/circ-policies-brief-view.component';
+import { DocumentsBriefViewComponent } from '../record/brief-view/documents-brief-view.component';
+import { LibraryComponent } from '../record/custom-editor/libraries/library.component';
+import { LibrariesBriefViewComponent } from '../record/brief-view/libraries-brief-view.component';
+import { PatronsBriefViewComponent } from '../record/brief-view/patrons-brief-view.component';
+import { PersonsBriefViewComponent } from '../record/brief-view/persons-brief-view.component';
+import { ItemTypesBriefViewComponent } from '../record/brief-view/item-types-brief-view.component';
+import { PatronTypesBriefViewComponent } from '../record/brief-view/patron-types-brief-view.component';
+import { of, Observable, Subscriber } from 'rxjs';
+import { TranslateService } from '@ngx-translate/core';
+import { I18nPluralPipe, NgLocaleLocalization } from '@angular/common';
+import { marker as _ } from '@biesbjerg/ngx-translate-extract-marker';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class RecordRoutingService {
+
+  constructor(
+    private router: Router,
+    private translateService: TranslateService
+  ) { }
+
+  initRoute() {
```

Comment:
Doc blocks for methods could be helpful

Reply:
A new task has been open: https://tree.taiga.io/project/rero21-reroils/task/1135?kanban-status=1224893